scripts/zones/Horlais_Peak/mobs/Sobbing_Eyes.lua

```diff
@@ -8,7 +8,7 @@ require("scripts/globals/status")
 local entity = {}
 
 entity.onMobInitialize = function(mob)
-    mob:setMod(xi.mod.SILENCERES, 75)
+    mob:setMod(xi.mod.SLEEP_MEVA, 75)
```

Comment:
Because down here they use base 100

Reply:
This is part of their new magic rewrite - we should not include any of this I believe